lib/logux/meta.rb

```diff
@@ -32,5 +32,31 @@ def time
     def id
       fetch('id')
     end
+
+    def undo_meta
+      self.class.new(undo_meta_hash)
+    end
+
+    private
+
+    RESEND_KEYS = %w[
+      users
+      nodes
+      reasons
```

Comment:
`reasons` is not part of resend keys. Every Logux node has own reasons (server reasons != client reasons).

Reply:
But I agree that `undo` action should have the same `reasons` as origin action (they must be both in the log). Maybe we can copy `reasons` from undid action in a different way?